### ISSUE TITLE
Unable to switch context because of directory not found error.

### ISSUE THREAD

**Issue by Alivers** [aabouzaid/kubech#9]
Firstly, Thanks for your effort to give the utility to mange multi-cluster at same time.
When I executed the change context command, the below error occurs.
```
aliver@dev-server:~$ kubechc arn:aws:eks:us-east-1:xxxxxx:cluster/MyCluster
touch: cannot touch '/home/aliver/.kube/config.dest.d/arn:aws:eks:us-east-1:xxxxxx:cluster/MyCluster-default': No such file or directory
chmod: cannot access '/home/aliver/.kube/config.dest.d/arn:aws:eks:us-east-1:xxxxxx:cluster/MyCluster-default': No such file or directory
-bash: /home/aliver/.kube/config.dest.d/arn:aws:eks:us-east-1:xxxxxx:cluster/MyCluster-default: No such file or directory
```
Seem it should `mkdir -p xxx` firstly to touch the new configuration.

**Comment by aabouzaid** [comment]
Hi @Alivers :wave:
Thanks for reporting this issue.

The source of the problem is that the context `arn:aws:eks:us-east-1:xxxxxx:cluster/MyCluster` has a forward slash,
so Kubech translates `MyCluster` as a file inside the `arn:aws:eks:us-east-1:xxxxxx:cluster` folder.

I will make a fix to remove any problematic characters when it creates the kube config file.

**Comment by Alivers** [comment]
> 

Thanks for your work. It is really useful in my daily work.

**Comment by aabouzaid** [comment]
Thanks @Alivers :rocket: 
Fixed by [9f11a54](https://github.com/aabouzaid/kubech/commit/9f11a5486152de8ea13ef0fc1e269187ecaa7db5)
